here-naksha-lib-handlers/src/main/java/com/here/naksha/lib/handlers/internal/AdminFeatureEventHandler.java

```diff
@@ -49,20 +51,23 @@ abstract class AdminFeatureEventHandler<FEATURE extends NakshaFeature> extends A
     this.featureClass = featureClass;
   }
 
-  /**
-   * The method invoked by the event-pipeline to process EventHandler specific read/write operations
-   *
-   * @param event the event to process.
-   * @return the result.
-   */
   @Override
-  public final @NotNull Result processEvent(@NotNull IEvent event) {
+  protected EventProcessingStrategy processingStrategyFor(IEvent event) {
+    Request<?> request = event.getRequest();
+    if (request instanceof ReadRequest || request instanceof WriteXyzFeatures) {
+      return PROCESS;
+    }
+    return SUCCEED_WITHOUT_PROCESSING;
```

Comment:
As a default behaviour, let's use NOT_IMPLEMENTED. Because, I think it will be better to get an error than getting false impression of success result.